fearless/Modules/Staking/YourValidators/ViewModel/YourValidatorsViewModelFactory.swift

```diff
@@ -28,52 +28,62 @@ final class YourValidatorsViewModelFactory {
             return balanceViewModeFactory.amountFromValue(amount)
         }()
 
+        let shouldHaveError: Bool = {
+            { if case .slashed = model.myNomination { return true }; return false }()
+        }()
+
         return YourValidatorViewModel(
             address: model.address,
             icon: icon,
             name: model.identity?.displayName,
             amount: amountTitle,
-            shouldHaveWarning: model.stakeInfo?.oversubscribed ?? false
+            shouldHaveWarning: model.stakeInfo?.oversubscribed ?? false,
+            shouldHaveError: shouldHaveError
         )
     }
 
     private func createTitle(
         for sectionStatus: YourValidatorsSectionStatus,
         count: Int
-    ) -> LocalizableResource<String> {
+    ) -> LocalizableResource<String?> {
```

Comment:
Looks like it is more natural to make LocalizableResource optional. There is no sense to apply localization to missing string

Reply:
Done...